lib/buffered_logger.rb

```diff
@@ -45,7 +41,17 @@ def started?
   end
 
   def sweep
-    @logdev.sweep
+    warn "[DEPRECATION] `sweep` is deprecated.  It is no longer necessary."
```

Comment:
We're the only consumers of this gem anyways so :+1:. Plus sweep wasn't really meant to be called manually, just randomly when a logger was ended.